### PR TITLE
backports/py-grpcio-testing: new aport

### PR DESCRIPTION
**Pull request by DaikiMaekawa**

### CODE REVIEW

**Comment by DaikiMaekawa** [comment]
@at-wat PTAL

this is required to merge https://github.com/seqsense/sq_basic_apps/pull/378

**Comment by DaikiMaekawa** [comment]
This should be merged soon. I opened the issue here. https://github.com/seqsense/aports-ros-experimental/issues/98

File: backports/py-grpcio-testing/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Contributor: Daiki Maekawa <daikimaekawa29@gmail.com>
+# Maintainer: Daiki Maekawa <daikimaekawa29@gmail.com>
+pkgname=py-grpcio-testing
+pkgver=1.20.0
+pkgrel=0
+pkgdesc="Testing utilities for gRPC Python."
+url="https://grpc.io/"
+arch="all"
+license="Apache License 2.0"
+makedepends="python2-dev py-setuptools cython-dev py2-grpcio py2-protobuf"
```
Comment (at-wat) [inline backports/py-grpcio-testing/APKBUILD]: `makedepends` is built time dependencies.
Last two should be `depends`. And version constraint should be specified.

Comment (DaikiMaekawa) [reply backports/py-grpcio-testing/APKBUILD]: Addressed

File: backports/py-grpcio-testing/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Contributor: Daiki Maekawa <daikimaekawa29@gmail.com>
+# Maintainer: Daiki Maekawa <daikimaekawa29@gmail.com>
+pkgname=py-grpcio-testing
+pkgver=1.20.0
+pkgrel=0
+pkgdesc="Testing utilities for gRPC Python."
+url="https://grpc.io/"
+arch="all"
+license="Apache License 2.0"
+depends="py-six>=1.5.2"
```
[the remark this answers is not in the document]
Comment (DaikiMaekawa) [reply backports/py-grpcio-testing/APKBUILD]: done

**Pull request merged**